.github/workflows/security.yml

```diff
@@ -59,6 +59,29 @@ jobs:
           path: /home/runner/vuln-cache
           key: trivy-cache-${{ github.run_id }}
 
+      - name: Get KEV list
+        run: |
+          cd /home/runner
+          curl -s -o kev.json https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json
+
+      - name: Cache KEV list
```

Comment:
That file is so cheap to download I don't think we even need to concern ourselves with caching it.